platform.js

```diff
@@ -130,7 +130,7 @@ module.exports = {
       return module.exports.spawn("taskkill /f /im javaw.exe");
     }
     else {
-      return module.exports.spawn("pkill -f " + module.exports.getJavaExecutablePath());
+      return module.exports.spawn("pkill -f Rise.*jar");
```

Comment:
Good point @ahmedalsudani .  In practice it won't matter because nothing is going to match the globbing so it will pass the result to pkill as expected.  But it's safer to wrap the argument in single quotes.